### PR TITLE
Add default device value for ie.LoadNetwork API

### PR DESCRIPTION
**Pull request by mashoujiang**
### Details:
 - *change LoadNetwork api with default deviceName=\"AUTO\"*

### Tickets:
 - *57530*

### CODE REVIEW

**Review by myshevts — APPROVED** [review]

File: inference-engine/tests/functional/plugin/shared/include/behavior/core_integration.hpp [hunk 1]
```diff
@@ -453,6 +453,12 @@ TEST_P(IEClassBasicTestP, ImportNetworkWithNullContextThrows) {
 // LoadNetwork
 //
 
+TEST_P(IEClassNetworkTestP, LoadNetworkWithoutDeviceNameNoThrow) {
+  SKIP_IF_CURRENT_TEST_IS_DISABLED()
+  Core ie;
+  ASSERT_NO_THROW(ie.LoadNetwork(actualNetwork));
```
Comment (ilya-lavrenov) [inline inference-engine/tests/functional/plugin/shared/include/behavior/core_integration.hpp:459]: currently, AUTO selects between GPU and CPU, how will it work on yocto?

Comment (mashoujiang) [reply inference-engine/tests/functional/plugin/shared/include/behavior/core_integration.hpp:459]: I guess that will throw exception, will the yocto case run this test? which plugin will the yocto run? (kmb-plugin?) not very clear with that usecase.. @wangleis

Comment (ilya-lavrenov) [reply inference-engine/tests/functional/plugin/shared/include/behavior/core_integration.hpp:459]: In this case `LoadNetwork` with no args does not work on all platforms.

Comment (mashoujiang) [reply inference-engine/tests/functional/plugin/shared/include/behavior/core_integration.hpp:459]: I see tests on cpuFuncTest and gpuFuncTest are passed, so what do you mean not workable on all platforms, could you please provide more info?

**Pull request closed without merging**